internal/pkg/describe/status.go

```diff
@@ -191,21 +193,68 @@ func (s *ServiceStatusDesc) HumanString() string {
 	fmt.Fprintf(writer, "  %s\t%s\n", "Task Definition", s.Service.TaskDefinition)
 	fmt.Fprint(writer, color.Bold.Sprint("\nTask Status\n\n"))
 	writer.Flush()
-	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", "ID", "Image Digest", "Last Status", "Health Status", "Started At", "Stopped At")
+	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", "ID", "Image Digest", "Last Status", "Started At", "Stopped At", "Health Status")
 	for _, task := range s.Tasks {
 		fmt.Fprint(writer, task.HumanString())
 	}
 	fmt.Fprint(writer, color.Bold.Sprint("\nAlarms\n\n"))
 	writer.Flush()
-	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\n", "Name", "Health", "Last Updated", "Reason")
+	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\n", "Name", "Condition", "Last Updated", "Health")
 	for _, alarm := range s.Alarms {
 		updatedTimeSince := humanizeTime(alarm.UpdatedTimes)
-		fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\n", alarm.Name, alarm.Status, updatedTimeSince, alarm.Reason)
+		printWithMaxWidth(writer, "  %s\t%s\t%s\t%s\n", maxColumnWidth, alarm.Name, alarm.Condition, updatedTimeSince, alarmHealthColor(alarm.Status))
+		fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\n", "", "", "", "")
 	}
 	writer.Flush()
 	return b.String()
 }
 
+func printWithMaxWidth(w *tabwriter.Writer, format string, width int, members ...string) {
+	memberSlices := make([][]string, len(members))
+	m := 0
+	for ind, member := range members {
+		var memberSlice []string
+		var res string
+		// https://stackoverflow.com/questions/25686109/split-string-by-length-in-golang
+		for i, r := range []rune(member) {
+			res += string(r)
+			if i > 0 && (i+1)%width == 0 {
+				memberSlice = append(memberSlice, res)
+				res = ""
+			}
+		}
+		if res != "" {
+			memberSlice = append(memberSlice, res)
+		}
+		m = int(math.Max(float64(len(memberSlice)), float64(m)))
+		memberSlices[ind] = memberSlice
+	}
+	for i := 0; i < m; i++ {
+		args := make([]interface{}, len(memberSlices))
```

Comment:
Why empty interface? can we not do `string` here

Reply:
discussed offline